integration/webhook_test.go

```diff
@@ -0,0 +1,175 @@
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	k8sApiErrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	addonsv1alpha1 "github.com/openshift/addon-operator/apis/addons/v1alpha1"
+	"github.com/openshift/addon-operator/integration"
+)
+
+const (
+	CATALOG_SOURCE_URL = "quay.io/osd-addons/reference-addon-index@sha256:58cb1c4478a150dc44e6c179d709726516d84db46e4e130a5227d8b76456b5bd"
+	ADDON_NAME         = "reference-addon-test-install-spec"
```

Comment:
Please make these constants lowerCamelCase - I don't think they need to be exported and other const declarations are also not upper snake case.